### DISCUSSION TITLE
2.0.3

### DISCUSSION THREAD

**Discussion by vegegoku** [DominoKit/domino-ui#970] [Releases]
### Enhancements and Features 🎉

* #686 Support Dropdown for responsive tabs.
* #774 Implement a date range picker component.
* #931 overflow tabs support.
* #951 Enhance how we append/remove Prefix/PostFix/primamry addons to element and enhance how we append multiple childs.
* #952 add TextArea support to DelayedTextInput.
* #954 DateBox should close the popup when cleared and call change listeners when value is null.
* #962 Add configuration for RichTextEditor buttons and toolbars.
* #963 Add a new Dialog size FULL.
* #964 Add more color variables variations for context context css styles.
* #966 add findTabByKey to tabs.
* #968 Slider bubble cuts off text.

### Fixes 🛠

* #953 Clearing the search in a grouped select does not restore all options.
* #955 Empty submenu opens as horizontal line.
* #959 Single Select Old Value is always null when change listener is called.
* #960 calling withValue(null) on a Select should clear it.
* #961 Wrong weekend day are marked as weekend.
* #967 Select sends incorrect oldvalue when clearing.
* #969 Selection listeners on ListGroup are not triggered.

**Full Changelog**: https://github.com/DominoKit/domino-ui/compare/2.0.2...2.0.3

<hr /><em>This discussion was created from the release <a href='https://github.com/DominoKit/domino-ui/releases/tag/2.0.3'>2.0.3</a>.</em>

**Comment by FrankHossfeld** [comment]
Great job! Already using!
